client/gutenberg/editor/calypsoify-iframe.jsx

```diff
@@ -134,14 +136,21 @@ class CalypsoifyIframe extends Component {
 
 	render() {
 		const { iframeUrl, siteId } = this.props;
-		const { isMediaModalVisible, allowedTypes, multiple } = this.state;
+		const { isMediaModalVisible, allowedTypes, multiple, isIframeLoaded } = this.state;
 
 		return (
 			<Fragment>
 				{ /* eslint-disable-next-line wpcalypso/jsx-classname-namespace */ }
 				<div className="main main-column calypsoify is-iframe" role="main">
-					{ /* eslint-disable-next-line jsx-a11y/iframe-has-title, wpcalypso/jsx-classname-namespace */ }
-					<iframe ref={ this.iframeRef } className={ 'is-iframe-loaded' } src={ iframeUrl } />
+					{ ! isIframeLoaded && <Placeholder /> }
+					{ /* eslint-disable-next-line jsx-a11y/iframe-has-title */ }
+					<iframe
+						ref={ this.iframeRef }
+						/* eslint-disable-next-line wpcalypso/jsx-classname-namespace */
+						className={ isIframeLoaded ? 'is-iframe-loaded' : undefined }
+						src={ iframeUrl }
+						onLoad={ () => this.setState( { isIframeLoaded: true } ) }
```

Comment:
Note that we generally avoid creating functions inside of JSX attributes because they get recreated on every render.

```js
markIframeLoaded = () => this.setState( { isIframeLoaded: true } );

…

<iframe … onLoad={ this.markIframeLoaded } … >
```